seeds/dbt_project_evaluator_exceptions.csv

```diff
@@ -1 +1,2 @@
-fct_name,column_name,id_to_exclude,comment
\ No newline at end of file
+fct_name,column_name,id_to_exclude,comment
+,,,
```

Comment:
Is this change necessary?

Reply:
Yes, absolutely. ClickHouse expects an empty line here and Bigquery fails if there are no commas to separate empty fields

Reply:
I've gave it a second thought